lms/templates/registration/account_activation_sidebar_notice.html

```diff
@@ -18,12 +17,12 @@ <h2 id="account-activation-title" class="account-activation-title sr">${_("Accou
         </span>
         <p class="status-note">${Text(_(
           "Check your {email_start}{email}{email_end} inbox for an account activation link from {platform_name}. "
-          "If you need help, contact {link_start}{platform_name} Support{link_end}."
+          "If you need help, see {link_start}support{link_end}."
         )).format(
+          platform_name=platform_name,
           email_start=HTML("<strong>"),
           email_end=HTML("</strong>"),
           email=email,
-          platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
           link_start=HTML("<a target='_blank' href='https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-'>"),
```

Comment:
Sure, give me some time and I'll get to this -- actually going to be doing something similar with ENT-370.